deployments/stacks/dpe-k8s/main.tf

```diff
@@ -57,3 +57,15 @@ module "synapse_dataset_to_crossiant_metadata" {
   enable_cors = true
 }
 
+module "synapse_data_catalog_minimal_jsonld" {
+  # This points to the prod cluster - The only location where we want to create the bucket
+  count  = var.cluster_name == "dpe-k8" ? 1 : 0
+  source      = "../../../modules/s3-bucket"
+  bucket_name = "synapse-data-catalog-jsonld"

```

Comment:
I think that this should include croissant in the name somewhere so it's explicit what these are. Following the name structure for the other bucket `synapse-croissant-metadata` we could just name this `synapse-croissant-metadata-minimal-content`, `synapse-croissant-metadata-minimal`, or a similar name structure.

Reply:
I like `synapse-croissant-metadata-minimal`. Let me fix that.